src/core/thread/mle.hpp

```diff
@@ -703,6 +704,15 @@ class Mle
     bool IsAnycastLocator(const Ip6::Address &aAddress) const;
 
     /**
+     * This method indicates whether or not an IPv6 address is a Mesh Local Prefix based address.
```

Comment:
nit: "is a Mesh Local Prefix based address" -> "is a Mesh Local Address"

same below.